packages/ui/src/icons/arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="11" height="12" viewBox="0 0 11 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M5.16667 11L5.16667 1M5.16667 1L1 5.16667M5.16667 1L9.33333 5.16667" stroke="#070709" style="stroke:#070709;stroke:color(display-p3 0.0275 0.0275 0.0353);stroke-opacity:1;" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
we should remove the hardcoded color from the svg to allow colors to be styled via css for theming